debootstrap/vendor_data.jinja

```diff
@@ -61,7 +95,7 @@
 		    'parent': 'jessie',
 		},
 		'wheezy': {
-		    'aliases': [ 'oldstable' ],
+		    'repository': 'archive',
```

Comment:
The other wheezy repositories also need to be modified to indicate "archive" or "archive-security" as their repository. Those that disappeared (wheezy-proposed-updates and wheezy-updates) should be dropped.

Reply:
Thanks for reviewing! Hopefully complete now.